hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java

```diff
@@ -1223,6 +1227,20 @@ protected boolean areSomeRegionReplicasColocated(Cluster c) {
     return false;
   }
 
+  protected final synchronized boolean idleRegionExist(Cluster c){
```

Comment:
`idleRegionExist` => `idleRegionServerExist` would be a better fit since we are checking if any RS is idle (without regions).